### ISSUE TITLE
npm module

### ISSUE THREAD

**Issue by Waog** [mbraude/TeddyMocks#2]
Hi,

making _TeddyMocks_ available as _npm module_ would be nice, for a fluent integration into standard workflows.

**Comment by JoshMcCullough** [comment]
Agreed, I'm not sure I'll use it unless it's available via a package manager. Appreciate the effort, though.
